frontend/public/index.html

```diff
@@ -22,9 +22,13 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Soundle</title>
+    <script src="https://cdn.bootcss.com/html2canvas/0.5.0-beta4/html2canvas.js"></script>
```

Comment:
What is the purpose of adding these additional scripts here instead of as an npm package? this isn't really consistent with everything else we've been doing on the project

Reply:
The script is useless, it's added when I was doing the share-score button.